0023-bilateral-transfer-protocol/0023-bilateral-transfer-protocol.md

```diff
@@ -468,12 +311,10 @@ Transfer ::= SEQUENCE {
 }
 ```
 
-`Transfer` is used to send money unconditionally on the bilateral ledger.
-Unlike a `Prepare`, `Transfer` is assumed to move funds immediately, without
-requiring a subsequent `Fulfill` call.
-
-Data to secure this payment (such as payment channel claims), can be attached
-under the protocol data.
+`Transfer` is used to send proof of payment, payment channel claims, or other
+settlement information to the other connector.
+The amount should indicate the relative value of this settlement state (compared to the
+previous settlement state), in a unit that was agreed out-of-band.
```

Comment:
Relative means this can be negative.

Is this description correct?

I thought this was used to carry a `sendMoney` which means the amount is always positive and the message is only sent by the connector that is making the transfer.

Reply:
Good point! I changed "relative" to "additional"